integration/sbom_test.go

```diff
@@ -165,6 +176,7 @@ func TestSBOM(t *testing.T) {
 			// Run "trivy sbom"
 			runTest(t, osArgs, tt.golden, outputFile, types.Format(tt.args.format), runOptions{
 				override: overrideFuncs(overrideSBOMReport, overrideUID, tt.override),
+				fakeUUID: tt.fakeUUID,
```

Comment:
It looks like we only use fakeUUID for `serialNumber`:
https://github.com/aquasecurity/trivy/blob/8b333099c454c1aa11296c0c910a15317814adb6/integration/testdata/fluentd-multiple-lockfiles-reused.cdx.json.golden#L5

This might be confusing (that we don't overwrite other UUID's).
Maybe we just don't check `serialNumber`?
@knqyf263 @afdesk wdyt?

Reply:
The serial number is generated by UUID. I think it's better to confirm it is generated as expected rather than ignoring it.